### DISCUSSION TITLE
Windows 95 protection error message

### DISCUSSION THREAD

**Discussion by MaximeLuk** [utmapp/UTM#5191] [UTM Help (macOS)]
When I install a windows 95 iso, it says "Windows protection error. You need to restart your device

**Configuration**  
* UTM Version: 4.1.6
* macOS Version: 13.2.1
* Mac Chip: M2 Pro

[config.plist.zip](https://github.com/utmapp/UTM/files/11074621/config.plist.zip)

**Comment by LostInCompilation** [comment]
Not a bug with UTM, check your configuration. I have both Windows95 and 98 running.

**Comment by tifasoftware** [comment]
try setting your CPU to pentium. If that doesn't work, look in to switching from i430vx to isapc. (that worked for me when installing nt 3.51)

**Reply by MaximeLuk** [reply]
Thanks a lot!! Let me try that

**Comment by MaximeLuk** [comment]
I tried switching to pentium CPU, same error. I tried switching from i440 pc to isapc, and I get "QEMU exited from an error: qemu-i386-softmmu: -device e1000e,mac=6E:96:D6:6A:B3:16,netdev=net0: No 'PCI' bus found for device 'e1000e'"

**Comment by rogurt123** [comment]
Try installing fix 95 cpu. replace the windows 95 iso with that one, boot into the fix 95 cpu iso, install the patcher, eject the cd, and reboot. worked for me when I tried.

**Comment by MaximeLuk** [comment]
How exactly do I do that?

**Reply by MaximeLuk** [reply]
oh, I found it

here is the link
[https://archive.org/details/fix-95-cpu-v3-final](https://archive.org/details/fix-95-cpu-v3-final)
